test/test.do

```diff
@@ -14,8 +14,8 @@ set trace off
 /* =================================================
    ==================== Check 1 ==================== 
    ================================================= */
-
-cd "check01"
+/*

```

Comment:
I believe you've commented out the first check, can you remove this?

Also can you change 

```Stata
cd "../check01"
```

back to 

```Stata
cd "check01"
```